contentcuration/contentcuration/viewsets/channel.py

```diff
@@ -262,13 +262,13 @@ def create(self, validated_data):
         validated_data["content_defaults"] = self.fields["content_defaults"].create(
             content_defaults
         )
+        instance = super(ChannelSerializer, self).create(validated_data)
         if "request" in self.context:
-            user_id = self.context["request"].user.id
+            user = self.context["request"].user
             # This has been newly created so add the current user as an editor
-            validated_data["editors"] = [user_id]
+            instance.editors.add(user)
             if bookmark:
-                validated_data["bookmarked_by"] = [user_id]
-        instance = super(ChannelSerializer, self).create(validated_data)
+                self.context["request"].user.bookmarked_channels.add(instance)
```

Comment:
Can also just use `user.bookmarked_channels.add(instance)` or `instance.bookmarked_by.add(user)` to be consistent with lines above